netci.groovy

```diff
@@ -180,7 +192,7 @@ static void addVsiMultiScm(def myJob, def project) {
                     relativeTargetDirectory('sdk')
                 }
                 // pull in a specific LKG commit from master.
-                branch('c82725bc657ad369ecd4e59bf860acf6205027b6')
+                branch('${GitBranchOrCommit}')
```

Comment:
This will now work, as GitBranchOrCommit points to the branch on project system repo. You need to use a specific LKG commit.